### PR TITLE
🙇‍♂️ 🙏 📿

### PR DESCRIPTION
**Pull request by jagarop**

### CODE REVIEW

File: src/bot.py [hunk 1]
```diff
@@ -359,6 +364,7 @@ def on_end_game(self):
     def on_end_run(self):
         success = self._char.tp_town()
         self._tps_left -= 1
+        self._pre_buffed = 0
```
Comment (aeon0) [inline src/bot.py:367]: this shouljd be in end_game(), no? end_run() is called after pindle and shenk routine.

**Comment by xsibit666** [comment]
Hello one question ist thos only Form difficultly hell?

**Comment by aeon0** [comment]
> Hello one question ist thos only Form difficultly hell?

This is a Pull Request. Not a "does this support feature x,y channel". Please ask your questions e.g. on the discord. thx.

**Review by jagarop — COMMENTED** [review]
yeah should be on end_game() and not in end_run() please change it, having issues to do it

**Review by aeon0 — APPROVED** [review]

**Comment by m88310012** [comment]
> 
> 
> Hello one question ist thos only Form difficultly hell?

you can modify the value(ex: 520) of hell_y in ui.ini.

**Pull request merged**